### ISSUE TITLE
About finetune on maniskill data

### ISSUE THREAD

**Issue by ketchup45** [thu-ml/RoboticsDiffusionTransformer#118]
I use the maniskill pt model and datasets which you provided on huggingface to finetune. However, when resuming training from one of my checkpoints (with --resume_from_checkpoint="latest"), only the loss of the first step is very small, maintaining the level of previous training. But in the following steps, the l2_loss will become particularly large again, reaching over 0.9, just like starting from 0 to retrain.
It reads: INFO - __main__ - Resuming training state failed. Attempting to only load from model checkpoint.
Have you experienced a similar situation before? I would appreciate it.

<img width="445" height="462" alt="Image" src="https://github.com/user-attachments/assets/bbcfec9e-919f-400e-a11d-a5e082a5acfc" />

**Comment by 18643638196xhx** [comment]
我在finetune.sh中添加了--resume_from_checkpoint="checkpoint-5000"，这对我有帮助。
